src/test/java/tests/applications/browser/AqualityServices.java

```diff
@@ -35,6 +40,13 @@ public static Injector getServiceProvider() {
         return getInstance().getInjector();
     }
 
+    public static <T extends AqualityModule<ChromeApplication>> void initInjector(T module) {
```

Comment:
is it fix of some bug? or how is it related with logger?

Reply:
it is related with tests cleanup. it's a test class.